src/video/windows/SDL_windowsmouse.c

```diff
@@ -521,6 +535,89 @@ static SDL_MouseButtonFlags WIN_GetGlobalMouseState(float *x, float *y)
     return result;
 }
 
+static void WIN_ApplySystemScale(void *internal, Uint64 timestamp, SDL_Window *window, SDL_MouseID mouseID, float *x, float *y)
+{
+    if (!internal) return;
```

Comment:
```suggestion
    if (!internal) {
        return;
    }
```
You've got a bunch of these, can you fix them to match the SDL coding style?

Thanks!